lib/pci.c

```diff
@@ -379,9 +405,15 @@ pci_config_space_access(vfu_ctx_t *vfu_ctx, char *buf, size_t count,
 
     while (count > 0) {
         vfu_region_access_cb_t *cb;
-        size_t size;
-
-        size = pci_config_space_next_segment(vfu_ctx, count, offset, &cb);
+        ssize_t size;
+
+        size = pci_config_space_next_segment(vfu_ctx, count, offset, is_write,
```

Comment:
could we keep it as size_t and just not set "cb" for the error condition maybe? what do you think?

Reply:
sure